src/components/CryptoSelectForm.js

```diff
@@ -1,20 +1,39 @@
-import React, { Component } from 'react';
+/*  I refactored this to be a sub component for the new SelectionForm component.
+		Once again I realized that to keep the code DRY I needed to refactor the selection
+		components to make them more reusable.  This now gets passed into Selection form
+		to render the proper content for the select form elements.
+		---The main reason why there is still this component and the IntlSelectForm is
+		because I couldn't get the crypto.name to render conditionally while being mapped.
+*/
+
+import React from 'react';
+import Radium from 'radium';
+import RadiumVars from '../RadiumVariables';
+
 
 const CryptoSelectForm = (props) =>{
 	return (
-		<div className="selectForm">
-      <p className="selectForm__label">Select Cryptocurrency</p>
-      <form onChange={props.setCrypto}>
-        <select className="selectForm__form">
-          {props.cryptoList.map(crypto => (
-            <option value={crypto.code} key={crypto.code}>
-              {crypto.name} - {crypto.code}
-            </option>
-          ))}
-        </select>
-      </form>
-		</div>
+		<select style={styles.selectForm__form}>
+			{props.list.map(crypto => (
+				<option key={crypto.code} value={crypto.code}>{crypto.name} – {crypto.code}</option>
+			))}
+		</select>
```

Comment:
This now just handles the mapping of crypto currencies inside the Select form element on the app.